src/wagtail_ai/static_src/image_description/main.tsx

```diff
@@ -0,0 +1,49 @@
+import './main.css';
+import { fetchResponse } from '../api';
+
+import wandIcon from '!!raw-loader!../wand_icon.svg';
+
+document.addEventListener('wagtail-ai:image-form', (event) => {
+  const input = event.target as HTMLInputElement;
+  const imageId = input.dataset['wagtailai-image-id']!;
+  const csrfToken = (
+    input.form!.querySelector(
```

Comment:
As far as I remember, you can specify a generic type on the querySelector.

`querySelector<HTMLInputElement>("input[name=csrfmiddlewaretoken]")`.

Reply:
TIL, thanks!